consensus/consensus_v2.go

```diff
@@ -711,6 +711,7 @@ func (consensus *Consensus) tryCatchup() error {
 			return err
 		}
 		select {
+		// TODO(jacky): check if this is still needed.
```

Comment:
It is still used in DNS sync. But removing will also not harm :-) BTW, I think the default case of this select can be downgraded to Trace since this channel is not used for stream sync.